src/views/base/scripts/heading-link-script.astro

```diff
@@ -0,0 +1,88 @@
+---
+import { Icon } from "astro-icon";
+import style from "./heading-link.module.scss";
+---
+
+<template id="heading-link-template">
+	<button class={style.headingLinkContainer} data-heading-anchor="copy">
+		<div class={style.headingLink}>
+			<span data-heading-link="copy">
+				<div class={style.icon} aria-hidden>
+					<Icon name="link" />
+				</div>
+				 Copy link
+			</span>
+			<span data-heading-link="copied" hidden>
+				<div class={style.icon} aria-hidden>
+					<Icon name="checkmark" />
+				</div>
+				 Link copied!
+			</span>
```

Comment:
We need to add another span with a `aria-live="polite"` that includes the `hidden` area but not the `data-heading-link="copy"` area so that the `copied!` can be read out loud

![image](https://github.com/unicorn-utterances/unicorn-utterances/assets/9100169/5b37a12f-c4a8-47d4-b35d-7477fdc0854d)